ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -99,9 +96,9 @@ export default Route.extend({
       aws: 'role-aws',
       // secret or secret-v2
       cubbyhole: 'secret',
```

Comment:
you can see more clearly now what model this flow uses and what engine uses it. The model is 'secret' and Cubbyhole, anything generic (kv1) and I believe transit still uses part of it.